app/src/org/commcare/dalvik/provider/CaseDataAPI.java

```diff
@@ -13,8 +15,7 @@
  *
  */
 public class CaseDataAPI {
-    
-    public static final String AUTHORITY = "org.commcare.dalvik.case";
+    public static final String AUTHORITY = BuildConfig.CC_AUTHORITY + ".case";
```

Comment:
BuildConfig.CC_AUTHORITY is set to ${applicationId}.provider, which in the normal CommCare case would be org.commcare.dalvik.provider, right? So that would make this not equivalent to what was there before.